#**Practical-8:**
**Name: Preeti Sahani**

In [ ]:
import pandas as pd

df=pd.read_csv('/content/stats.csv')
df

Name  Salary Country
0        Dan   40000     USA
1  Elizabeth   32000  Brazil
2        Jon   45000   Italy
3      Maria   54000     USA
4       Mark   72000     USA
5       Bill   62000  Brazil
6       Jess   92000   Italy
7      Julia   55000     USA
8       Jeff   35000   Italy
9        Ben   48000  Brazil

#**Measure of Central Tendency**

In [ ]:
# Mean Salary
mean1=df['Salary'].mean()
mean1

53500.0

In [ ]:
# Sum of Salaries
sum1=df['Salary'].sum()
sum1

535000

In [ ]:
# Maximum Salary
max1=df['Salary'].max()
max1

92000

In [ ]:
# Minimum Salary
min1=df['Salary'].min()
min1

32000

In [ ]:
# Total Count
count1=df['Salary'].count()
count1

10

In [ ]:
# Median
median=df['Salary'].median()
median

51000.0

In [ ]:
# Mode
mode1=df['Salary'].mode()
mode1

0    32000
1    35000
2    40000
3    45000
4    48000
5    54000
6    55000
7    62000
8    72000
9    92000
dtype: int64

In [ ]:
countrywise_sum=df.groupby(['Country']).sum()
countrywise_sum

Salary
Country        
Brazil   142000
Italy    172000
USA      221000

In [ ]:
countrywise_count=df.groupby(['Country']).count()
countrywise_count

Name  Salary
Country              
Brazil      3       3
Italy       3       3
USA         4       4

#**Measure of Variability**

In [ ]:
# Variance of salaries
var1=df['Salary'].var()
var1

332055555.5555556

In [ ]:
# Standard deviation
std1=df['Salary'].std()
std1

18222.391598128816

#**Measure of Symmetry**

In [ ]:
skew1=df.skew(axis=0, skipna=True)
skew1

Salary    1.021551
dtype: float64

**The Skewness is positive so x will have right side tail.**

#**Covariance and Correlation**

In [ ]:
import pandas as pd
bw=pd.read_csv('/content/BirthWeight.csv')
bw.head()

Infant ID  Gestational Age (Weeks)  Birth Weight (Grams)
0          1                     34.7                  1895
1          2                     36.0                  2030
2          3                     29.3                  1440
3          4                     40.1                  2835
4          5                     35.7                  3090

In [ ]:
bw.set_index('Infant ID', inplace=True)
bw.head()

Gestational Age (Weeks)  Birth Weight (Grams)
Infant ID                                               
1                             34.7                  1895
2                             36.0                  2030
3                             29.3                  1440
4                             40.1                  2835
5                             35.7                  3090

In [ ]:
print(bw.cov())

print(bw.corr(method="pearson"))

                         Gestational Age (Weeks)  Birth Weight (Grams)
Gestational Age (Weeks)                 9.963824              1798.025
Birth Weight (Grams)                 1798.025000            485478.750
                         Gestational Age (Weeks)  Birth Weight (Grams)
Gestational Age (Weeks)                 1.000000              0.817519
Birth Weight (Grams)                    0.817519              1.000000


**Covariance indicates that there is correlation exists between two Correlation coefficient of 0.818 indicates the relation between two variable is strong.**